METIS/METIS_DET_IMG_N_Aquarius.yaml

```diff
@@ -6,9 +6,12 @@ name: metis_detector_array
 description: Aquarius detector for METIS IMG-NQ
 
 properties:
-    temperature: -265
-    dit: "!OBS.dit"
-    ndit: "!OBS.ndit"
+  image_plane_id: 0
+  temperature: -265
+  dit: "!OBS.dit"
+  ndit: "!OBS.ndit"
+  mindit: 0.008            # seconds, E-REP-NOVA-MET-1191, v2-0
+  full_well: !!float 1.e7  # electrons, E-REP-NOVA-MET-1191, v2-0
```

Comment:
For the Aquarius, we had 2 full wells (for the GeoSnap as well actually), one for the high capacity mode (in Aquarius lingo called "low gain") of ~1e7 [e-/pix] and one for the low capacity mode ("high gain") of 1e6 [e-/pix]. We were planning to use the low capacity mode generally, since operating at illumination levels requiring the high capacity mode, the Aquarius suffered badly from ELFN (Excess Low Frequency Noise), killing the sensitivity.

Reply:
Changed to 1e6.